composeApp/src/commonMain/kotlin/org/ooni/probe/ui/shared/NotificationMessages.kt

```diff
@@ -0,0 +1,33 @@
+package org.ooni.probe.ui.shared
+
+import androidx.compose.material3.SnackbarResult
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import ooniprobe.composeapp.generated.resources.AddDescriptor_Toasts_Installed
+import ooniprobe.composeapp.generated.resources.LoadingScreen_Runv2_Canceled
+import ooniprobe.composeapp.generated.resources.LoadingScreen_Runv2_Failure
+import ooniprobe.composeapp.generated.resources.Res
+import org.jetbrains.compose.resources.stringResource
+import org.ooni.probe.LocalSnackbarHostState
+import org.ooni.probe.data.models.SnackBarMessage
+
+@Composable
+fun NotificationMessages(
+    message: List<SnackBarMessage>,
+    onMessageDisplayed: (SnackBarMessage) -> Unit = { },
```

Comment:
This argument is not being used, meaning there's a part missing in the messages queue.

The ViewModel needs to append messages to the list/queue, and when the ViewModel receives the event of a message being displayed, it needs to remove the message from the list/queue.

This is important because otherwise, if the view gets recreated, it will show the last message shown again, because it wasn't taken off the list. And if 2 messages are triggered at the same time from the ViewModel, one will override the other, while in a queue, they are shown in order.